Description: Link sqlite provider against sqlite3
 The sqlite provider uses dlopen() to load the sqlite3 library.
 As a consequence, libgda-5.0-4 did not have a dependency on libsqlite3-0
 so it was not guaranteed that libsqlite3-0 was actually installed when
 applications, like anjuta, were using the sqlite provider.
Author: Cyril Brulebois <kibi@debian.org>
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=667751
--- libgda5-5.0.3.orig/providers/sqlite/Makefile.am
+++ libgda5-5.0.3/providers/sqlite/Makefile.am
@@ -18,6 +18,7 @@ libgda_sqlite_la_SOURCES = \
 libgda_sqlite_la_LDFLAGS = -export-dynamic -module -avoid-version $(NO_UNDEFINED) $(LIBTOOL_PROV_EXPORT_OPTIONS)
 libgda_sqlite_la_LIBADD = \
 	$(top_builddir)/libgda/libgda-5.0.la \
+	$(SQLITE_LIBS) \
 	$(COREDEPS_LIBS)
 
 xmldir   = $(datadir)/libgda-5.0
